### PR TITLE
Use coq_makefile to build flocq.dev

### PR DESCRIPTION
**Pull request by SkySkimmer**
This makes it possible to get per file timings in Coq's bench (which sets the environment variable TIMING=1)
I think this works OK but I'm not 100% sure we want to merge this, what do other people think?

cc @silene as maintainer according to the opam file

### CODE REVIEW

**Comment by silene** [comment]
Actually, I don't maintain of that file. I guess whoever committed it just copy-pasted the one from `released` (which I maintain), without changing that information.

File: extra-dev/packages/coq-flocq/coq-flocq.dev/opam [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+maintainer: "everybody"
 homepage: "http://flocq.gforge.inria.fr/"
 dev-repo: "git+https://scm.gforge.inria.fr/anonscm/git/flocq/flocq.git"
 license: "LGPL 3"
 build: [
   ["./autogen.sh"]
   ["./configure"]
-  ["./remake" "-j%{jobs}%"]
+  ["find" "src" "-name" "*.v" "-fprint" "vfiles"]
+  ["sh" "-c" "cat _CoqProject vfiles > _CoqProject.full"] # how else to add to a file in opam?
```
Comment (gares) [inline extra-dev/packages/coq-flocq/coq-flocq.dev/opam]: https://opam.ocaml.org/doc/Manual.html#opam
Look for `patches:`, `extra-files:` and `extra-sources:`.

Your solution is also ok. You can also use `coq_makefile -o Makefile -f _CoqProject more files here`.

**Comment by palmskog** [comment]
This currently looks like a hack that I'm personally not inclined to merge. Also, setting maintainer to `everybody` is nonsensical. Project build script changes using opam-hosted patches would be a much better way to address this than having a long sequence of build commands in the `opam` file.

**Comment by gares** [comment]
@SkySkimmer do you plan to apply the requested changes, namely:
- reasonable maintainer field
- patch the software with an actual patch

**Pull request closed without merging**